pages/projects/digital-center.js

```diff
@@ -0,0 +1,288 @@
+              src={t("dc:Concept2Img2")}
+            >
+              <p className="text-sm">{t("dc:Concept2Img2Caption")}</p>
+              <HTMList
+                tag="ol"
+                listClassName={"ml-6 mt-2 text-sm list-decimal list-outside"}
+                liClassName={"mt-1"}
+                content={t("dc:Concept2Img2CaptionList")}
+              />
+            </ThumbnailWithCaption>
+            <ThumbnailWithCaption
+              title={t("dc:Concept2Img3Title")}
+              alt={t("dc:Concept2Img3Alt")}
+              src={t("dc:Concept2Img3")}
+            >
+              <p className="text-sm">{t("dc:Concept2Img3Caption")}</p>
+              <HTMList
+                tag="ol"
+                listClassName={"ml-6 mt-2 text-sm list-decimal list-outside"}
+                liClassName={"mt-1"}
+                content={t("dc:Concept2Img3CaptionList")}
+              />
+            </ThumbnailWithCaption>
+          </div>
+        </div>
+
+        <h2 className="text-lg mt-10">{t("dc:Concept3Heading")}</h2>
+        <p className="text-base mt-6">{t("dc:Concept3P1")}</p>
+        <HTMList
+          tag="ol"
+          listClassName={"text-base mt-4 ml-8 list-decimal list-outside"}
+          liClassName={"mt-1"}
+          content={t("dc:Concept3P1List")}
+        />
+
+        <div className="mx-auto">
+          <div className="mt-6 grid grid-cols-1 lg:grid-cols-2 items-start gap-6">
+            <ThumbnailWithCaption
+              title={t("dc:Concept3Img1Title")}
+              alt={t("dc:Concept3Img1Alt")}
+              src={t("dc:Concept3Img1")}
+            >
+              <p className="text-sm">{t("dc:Concept3Img1Caption")}</p>
+              <HTMList
+                tag="ol"
+                listClassName={"ml-6 mt-2 text-sm list-decimal list-outside"}
+                liClassName={"mt-1"}
+                content={t("dc:Concept3Img1CaptionList")}
+              />
+            </ThumbnailWithCaption>
+          </div>
+        </div>
+
+        <h2 className="text-lg mt-10">{t("dc:Concept4Heading")}</h2>
+        <p className="text-base mt-6">{t("dc:Concept4P1")}</p>
+
+        <div className="mx-auto">
+          <div className="mt-6 grid grid-cols-1 lg:grid-cols-2 items-start gap-6">
+            <ThumbnailWithCaption
+              title={t("dc:Concept4Img1Title")}
+              alt={t("dc:Concept4Img1Alt")}
+              src={t("dc:Concept4Img1")}
+            >
+              <p className="text-sm">{t("dc:Concept4Img1Caption")}</p>
+              <HTMList
+                tag="ol"
+                listClassName={"ml-6 mt-2 text-sm list-decimal list-outside"}
+                liClassName={"mt-1"}
+                content={t("dc:Concept4Img1CaptionList")}
+              />
+            </ThumbnailWithCaption>
+          </div>
+        </div>
+      </section>
+
+      <CallToAction
+        title={t("signupTitle")}
+        description={t("becomeAParticipantDescription")}
```

Comment:
We are passing in an html string here so we use the `html` prop instead of the `description` prop. I've noticed that we will need to format the string as well to increase the font size on mobile, I've created a task for that in Trello.

Reply:
fixed!